Source/Core/AI/JobDrivers/JobDriver_JailorDeliverFood.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using RimWorld;
+using Unity.Jobs.LowLevel.Unsafe;
+using Verse;
+using Verse.AI;
+
+namespace PrisonLabor.Core.AI.JobDrivers
+{
+    public class JobDriver_JailorDeliverFood : JobDriver_FoodDeliver
+    {
+
+        protected override IEnumerable<Toil> MakeNewToils()
+        {
+            var toils = base.MakeNewToils().ToList();
+
+            toils.RemoveRange(toils.Count - 1, 1);
+
+            Toil toil2 = new Toil();
+            toil2.initAction = delegate
+            {
+                pawn.carryTracker.TryDropCarriedThing(toil2.actor.CurJob.targetC.Cell, ThingPlaceMode.Direct, out Thing food, placedAction: (a, b) =>
+                {
+                    a.SetForbidden(true);
+                    if (((Pawn)TargetB).needs.food.CurLevelPercentage < 0.15)
+                    {
+                        var def = new JobDef
```

Comment:
And why this isnt in defs?

Reply:
Right, I forgot and i was testing the concept.

Reply:
Btw why you add deliver food for jailors when it is warden's job?